src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/HtmlDoclet.java

```diff
@@ -342,8 +343,18 @@ private void copyLegalFiles(boolean includeJQuery) throws DocletException {
         String legalNotices = configuration.getOptions().legalNotices();
         switch (legalNotices) {
             case "", "default" -> {
-                Path javaHome = Path.of(System.getProperty("java.home"));
-                legalNoticesDir = javaHome.resolve("legal").resolve(getClass().getModule().getName());
+                // use a dummy resource as a stand-in, because we cannot get the URL for a resources directory
+                var url = HtmlDoclet.class.getResource(DocPaths.RESOURCES.resolve(DocPaths.STYLESHEET).getPath());
```

Comment:
Have you considered getting the legal notices directory from the parent of `resources/legal/jquery.md` instead of `resources/stylesheet.css`?   Using a relevant file would help the reader.

Reply:
Noted. Even though we may not need to copy the file to the output, we can still always use it to find the directory.